### PR TITLE
Customer Credit API Fix Spectral errors

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [X] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to fix spectral errors (Customer Credit API) - EDU 12754

### CODE REVIEW

File: VTEX - Customer Credit API.json [hunk 1]
```diff
@@ -1162,7 +1162,7 @@
                                             "creditLimit": 2000.0,
                                             "updatedAt": "2020-01-02T15:26:40.3425717Z",
                                             "createdAt": "2019-12-04T14:06:02.7905117Z",
-                                            "description": "",
+                                            "description": "This field is deprecated and is returned as an empty string",
```
Comment (Isabella-Veloso) [inline VTEX - Customer Credit API.json:1165]:
```suggestion
                                            "description": "This field is deprecated and is returned as an empty string.",
```

File: VTEX - Customer Credit API.json [hunk 2]
```diff
@@ -1178,7 +1178,7 @@
                                             "creditLimit": 1.0,
                                             "updatedAt": "2020-05-08T19:16:40.7202156Z",
                                             "createdAt": "2020-05-08T19:16:40.5951631Z",
-                                            "description": "",
+                                            "description": "This field is deprecated and is returned as an empty string",
```
Comment (Isabella-Veloso) [inline VTEX - Customer Credit API.json:1181]:
```suggestion
                                            "description": "This field is deprecated and is returned as an empty string.",
```

**Review by Isabella-Veloso — CHANGES_REQUESTED** [review]
Adicionar ponto final em todas as descrições.

File: VTEX - Customer Credit API.json (hunk 2 above)
Comment (ricardoaerobr) [reply VTEX - Customer Credit API.json]: Corrigido.

**Review by Isabella-Veloso — APPROVED** [review]
LGTM

**Pull request merged**